app/inspect/index.tsx

```diff
@@ -16,7 +22,7 @@ import { useNavigation } from '@react-navigation/native'
 const Index = () => {
   const faceDetectionOptions = useRef<FaceDetectionOptions>({ performanceMode: 'fast' }).current
   const camera = useRef<Camera>(null)
-  const device = useCameraDevice('back')
+  const device = useCameraDevice('front')
```

Comment:
Could you clarify the reason for switching to the front camera? The app requires using the back camera for the inspect feature, so we should revert to using `useCameraDevice('back')` unless there’s a specific need for this change.

Reply:
For testing purposes during development, we've opted to use the front camera as it's easier to show the face and test functionality. We'll switch to the back camera in the final version.